src/map_ref.rs

```diff
@@ -154,6 +154,14 @@ where
         self.map.insert(key, value, &self.guard)
     }
 
+    /// Maps `key` to `value` in this table if, and only if, `key` is not
+    /// already mapped to a value.
+    /// See also [`HashMap::try_insert`].
```

Comment:
It'd be great if we could leave an empty line after the first sentence of the docs. This makes it a little easier to visually parse.

Reply:
No problem.